@types/resolve-promise-cb.d.ts

```diff
@@ -0,0 +1 @@
+export function resolvePromiseCb<T>(err?: Error, res?: T, cb?: (err?: Error, res?: T) => void): void|Promise<T>
```

Comment:
sorry my mistake, but this might be better like this for type detection in `@types/resolve-promise-cb.d.ts`:
```ts
export function resolvePromiseCb<T>(err?: Error, res?: T): Promise<T>
export function resolvePromiseCb<T>(err?: Error, res?: T, cb: (err?: Error, res?: T) => void): void
```

Reply:
fixed